src/core_atmosphere/physics/mpas_atmphys_driver_lsm.F

```diff
@@ -717,6 +717,7 @@ subroutine lsm_to_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
  if(config_frac_seaice) then
     do j = jts,jte
     do i = its,ite
+!$acc update host(chs_sea,chs2_sea,cqs2_sea,cpm_sea,hfx_sea,lh_sea,qfx_sea,qgh_sea,qsfc_sea,tsk_sea)
```

Comment:
the update host directive is inside the loop. Is there a reason?

Reply:
@mgduda 
The file-diff on the "Files changed" page shows it as being outside the loop.
My copy of the directory still shows the problem being in the file.
I don't know how to deal with this discrepancy.